circle.yml

```diff
@@ -0,0 +1,54 @@
+machine:
+  node:
+    version: 0.10.48
+  environment:
+    browser_name: $(case $CIRCLE_NODE_INDEX in 0) browser='chrome' ;; 1) browser='firefox' ;; 2) browser='internet explorer' ;; 3) browser='iphone' ;; esac; echo $browser)
+    browser_version: $(case $CIRCLE_NODE_INDEX in 3) version='8.2' ;; esac; echo $version)
+    SAUCE_LOG: $CIRCLE_ARTIFACTS/sauce_log.txt
+    WATAI_TESTS: test/integration
+    SAUCE_USERNAME: mes-aides-bot
+    # SAUCE_ACCESS_KEY is defined in the CircleCI web UI
+
+dependencies:
+  post:
+    - npm install saucelabs@0.1.1 # allow sending Watai results to SauceLabs
+    - wget https://saucelabs.com/downloads/sc-latest-linux.tar.gz
+    - tar -xzf sc-latest-linux.tar.gz
+    - sudo pip install --upgrade pip
+    - sudo pip install gunicorn
+    - npm run install-openfisca
+
+test:
+  pre:
+    - npm test
```

Comment:
This shouldn't be in the `pre` section: this is an actual test, not a preparation of the test.